qiita_db/support_files/patches/14.sql

```diff
@@ -0,0 +1,2 @@
+--Placeholder for python to run
```

Comment:
Haha, either git or the conflict resolution got messed, this change should have only gone in 15.sql ... I'll let it slide :laughing: